AddOns/BasicFinancials/app/src/codeunit/AssistedSetupBF.Codeunit.al

```diff
@@ -0,0 +1,46 @@
+codeunit 20605 "Assisted Setup BF"
```

Comment:
Started to test and the wizard page looks a bit weird
![image](https://user-images.githubusercontent.com/34042170/85686956-b6876200-b6d0-11ea-807e-0cd61b1a3bba.png)

This is how it usually looks like
![image](https://user-images.githubusercontent.com/34042170/85687080-d454c700-b6d0-11ea-9aef-8d696bbcc498.png)

Reply:
I was told that I should use "AMC Banking 365 Fundamentals" as a reference when creating the Assisted setup, I think it looks pretty much alike

![image](https://user-images.githubusercontent.com/47662057/85688880-745f2000-b6d2-11ea-9d8a-92b7ceb58247.png)